### PR TITLE
Add Details Of User Guide (FAQ and Command Summary)

### PR DESCRIPTION
**Pull request by yuqitanyq**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -174,19 +174,15 @@ _Details coming soon ..._
...
-Action | Format, Examples
---------|------------------
-**Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
-**Clear** | `clear`
-**Delete** | `delete INDEX`<br> e.g., `delete 3`
-**Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
-**Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
-**List** | `list`
-**Help** | `help`
+| Category    | Format                                                                                                                                                                                                                                                                                                         |
```
Comment (yl-ang) [inline docs/UserGuide.md:184]: The table format can be further broken down into 3 different categories.

**Review by yl-ang — CHANGES_REQUESTED** [review]
Changes to make to the command summary table.
See the comment in line 184 for the proposed changes.

**Comment by yuqitanyq** [comment]
I have changed the Command Summary as proposed. However, I am not sure if the last command of the General commands is considered a command, as it is about clicking the up and down arrows, not typing in commands. Maybe we can discuss this as a team.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -174,19 +174,35 @@ _Details coming soon ..._
...
+| **Find a particular patron** | `patron find n/<name keyword> [n/<name keyword> ...]`                       |
+| **List all patrons**         | `patron list`                                                               |
+### Category: Book Commands
+| Function                  | Format Of Command                                                              |
+|---------------------------|--------------------------------------------------------------------------------|
+| **Add a new book**        | `book add n/<name> i/<ISBN> [a/<author> …] [t/<category tag> …]`               |
+| **Edit a current book**   | `book edit INDEX n/<name> i/<ISBN> [a/<author> …] [t/<category tag> …]`        |
+| **Delete a current book** | `book delete INDEX`                                                            |
+| **Borrow a book**         | `borrow b/<index> p/<index> d/<return date>`                                   |
+| **Return a book**         | `return [b/<index>] or [p/<index>]`                                            |
```
Comment (yl-ang) [inline docs/UserGuide.md:199]: Not sure if we should merge b/index or p/index as one bubble for both the borrow and return command in the user guide.

There could be ambiguity in [index](https://nusu-my.sharepoint.com/:w:/r/personal/e0665491_u_nus_edu/Documents/Document.docx?d=w71ddd0adbc1c418fa7c2009a26c295fb&csf=1&web=1&e=G8Jed5&nav=eyJjIjo4NjcxNjAwMDB9).

If our user runs the command `return p/<index>`, is the index preferring to the index from the patron list or the book list?

Maybe we can add borrow and return for patron and for book respectively?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -174,19 +174,35 @@ _Details coming soon ..._
...
+| **Request a book**        | `request INDEX1 INDEX2`                                                        |
+| **List all books**        | `book list`                                                                    |
+### Category: General Commands
+| Function                                   | Format Of Command                   |
+|--------------------------------------------|-------------------------------------|
+| **Clear the sample data**                  | `clear`                             |
+| **Exit the application**                   | `exit`                              |
+| **Quick guide on how to use commands**     | `help`                              |
+| **Navigating the different patrons/books** | **Clicking the up and down arrows** |
```
Comment (yl-ang) [inline docs/UserGuide.md:208]: LGTM!

File: docs/UserGuide.md [hunk 4]
```diff
@@ -174,19 +174,35 @@ _Details coming soon ..._
...
+| **Edit a current patron**    | `patron edit INDEX [n/<name>] [s/<id>] [p/<phone>] [e/<email>] [t/<tag> …]` |
+| **Delete a current patron**  | `patron delete INDEX`                                                       |
+| **Find a particular patron** | `patron find n/<name keyword> [n/<name keyword> ...]`                       |
+| **List all patrons**         | `patron list`                                                               |
+### Category: Book Commands
+| Function                  | Format Of Command                                                              |
+|---------------------------|--------------------------------------------------------------------------------|
+| **Add a new book**        | `book add n/<name> i/<ISBN> [a/<author> …] [t/<category tag> …]`               |
+| **Edit a current book**   | `book edit INDEX n/<name> i/<ISBN> [a/<author> …] [t/<category tag> …]`        |
+| **Delete a current book** | `book delete INDEX`                                                            |
```
Comment (yl-ang) [inline docs/UserGuide.md:197]: LGTM!

**Review by yl-ang — CHANGES_REQUESTED** [review]
Overall LGTM!

There might be some concern for the borrow and return commands under Book, we can review them over in the team meeting and make the changes

**Review by bingsen0806 — APPROVED** [review]
LGTM! 👍

**Review by yl-ang — APPROVED** [review]

**Pull request merged**